client/landing/stepper/hooks/use-site-copy.tsx

```diff
@@ -69,7 +70,8 @@ export const useSiteCopy = (
 		return siteFeatures ? siteFeatures : { isRequesting: true };
 	} );
 	const isAtomic = useSelect(
-		( select ) => site && options.enabled && select( SITE_STORE ).isSiteAtomic( site?.ID ),
+		( select ) =>
+			site && options.enabled && ( select( SITE_STORE ) as SiteSelect ).isSiteAtomic( site?.ID ),
```

Comment:
Theoretically, this might need `site` and not `site.ID` as a dependency for the result to always be up to date. Will update.

Reply:
Good point 👍